index.bs

```diff
@@ -681,68 +714,80 @@ credentials are acceptable to it. The user agent and/or platform locates credent
 the user to pick one that the script will be allowed to use. The user may choose not to provide a credential even if one is
 present, for example to maintain privacy.
 
-Note: This algorithm is synchronous; the {{Promise}} resolution/rejection is taken care of by
+Note: This algorithm is synchronous; the {{Promise}} resolution/rejection is handled by
 {{CredentialsContainer/get()|navigator.credentials.get()}}.
 
-<div class="note">
-    This method takes the following parameters:
+This method accepts a single argument:
 
-    :   |options|
-    ::  A {{CredentialRequestOptions}} object, containing a challenge that the selected authenticator is expected to
-        sign to produce the assertion, and additional options as described in [[#assertion-options]]
-</div>
+<dl dfn-type="argument" dfn-for="PublicKeyCredential/[[DiscoverFromExternalSource]](options)">
+    :   <dfn>options</dfn>
+    ::  This argument is a  {{CredentialRequestOptions}} object whose |options|["{{CredentialRequestOptions/publicKey}}"] member
```

Comment:
hm, I'm not sure who wrote `|options|["{{CredentialRequestOptions/publicKey}}"] member` -- possibly @mikewest as part of PR #384 et al  (?) .... so I've just been leaving that alone since I'm not yet an Adept.  in source form would it be `|options|.{{CredentialRequestOptions/publicKey}}` ?